sdk-java/src/main/java/ly/count/sdk/java/internal/Device.java

```diff
@@ -132,52 +126,23 @@ public Params buildMetrics() {
     /**
      * Get total RAM in Mb
      *
-     * @return total RAM in Mb or null if cannot determine
+     * @return total RAM in Mb
      */
     public Long getRAMTotal() {
```

Comment:
I did not put os bean as a global parameter, but maybe we can use this to get also os version and os name. It is more precise for calculating ram and getting os related things. And also it is os friendly, it can work both linux and non-linux systems. So we do not need additional check for different oses